spark/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM jupyter/pyspark-notebook:spark-3.1.1
+LABEL maintainer "peloton@lal.in2p3.fr"
+ENV REFRESHED_AT 2021-06-08
+
+# Add repo
+WORKDIR /home/jovyan/work
+ADD . .
+ENV PYTHONPATH=$PYTHONPATH:/home/jovyan/work/python:${SPARK_HOME}/python/test_coverage
+ENV PYTHONPATH=$PYTHONPATH:${SPARK_HOME}/python:${SPARK_HOME}/python/lib/py4j-0.10.9-src.zip
+ENV COVERAGE_PROCESS_START=/home/jovyan/work/.coveragerc
+
+USER root
+RUN apt-get update \
+     && apt-get install -y vim
+USER jovyan
+
+# Add dependencies
+RUN pip install -r requirements.txt
+RUN python config_rise.py
+
+# This ARG is passed during the build
+ARG PYSPARK_DRIVER_PYTHON
```

Comment:
Why does this need to be passed at build time if it is just an environment variable?

Reply:
Thanks - this is a leftover from a previous version. Note the PR is still WIP, and not intended for code review - so you may still find a lot of stupid things left.

Reply:
https://github.com/escape2020/school2021/pull/55/commits/e97457ab693eec2f9f9529ba1ade8b29ad2ecc8e